scanner/config.go

```diff
@@ -91,7 +93,15 @@ func Config(searchDir string) models.ScanResultModel {
 	// Setup
 	currentDir, err := os.Getwd()
 	if err != nil {
-		result.AddError("general", fmt.Sprintf("Failed to expand current directory path, error: %s", err))
+		result.AddErrorWithRecommendation("general", models.ErrorWithRecommendations{
+			Error: fmt.Sprintf("Failed to expand current directory path, error: %s", err),
+			Recommendations: step.Recommendation{
+				errormapper.DetailedErrorRecKey: errormapper.DetailedError{
+					Title: "Failed to expand current directory path",
```

Comment:
this should be the generic detailed error: https://github.com/bitrise-io/bitrise-init/pull/189/files#diff-1c05f132f2d789a26a34596e8993fe0e551b63d8359cc18f32f4bf5993cd0f0bR45